eng/pipelines/templates/steps/analyze.yml

```diff
@@ -43,6 +43,14 @@ steps:
     failOnStderr: true
     workingDirectory: 'sdk/${{parameters.ServiceDirectory}}'
 
+  - script: |
+      echo scanning copyright headers in $(pwd)
+      grep -L -r --include LICENSE -P "Copyright (\d{4}|\(c\)) Microsoft" . | tee >&2
```

Comment:
Is this needed in every module? Is the license in the root of the repo enough?

Reply:
Yes I believe every module needs to have a copy of the license.

Reply:
Hmmm, thinking about this again I think it only needs to be in the root of each module